packages/gestalt/src/Masonry.tsx

```diff
@@ -471,7 +493,9 @@ export default class Masonry<T> extends ReactComponent<Props<T>, State<T>> {
           height: layoutNumberToCssDimension(height),
         }}
       >
-        {renderItem({ data: itemData, itemIdx: idx, isMeasuring: false })}
+        {isLoading && _renderLoadingStateItems
+          ? _renderLoadingStateItems({ data: itemData as U, itemIdx: idx, isMeasuring: false })
```

Comment:
feels odd that `_renderLoadingstateItems` has the same API signature as `renderItem` since `isMeasuring` will always be false

Reply:
updated!